test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/Microsoft.AspNetCore.Diagnostics.FunctionalTests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
+    <SignAssembly>false</SignAssembly>
+    <PublicSign>false</PublicSign>
```

Comment:
Yeah it's required because this test project depends on samples that are unsigned.